features/upgrade/sdn/ipsec-upgrade.feature

```diff
@@ -0,0 +1,95 @@
+    Given the env is using "OVNKubernetes" networkType
+    And the IPsec is enabled on the cluster
+    Given I store all worker nodes to the :workers clipboard
+    Given I switch to cluster admin pseudo user
+    When I run the :new_project client command with:
+      | project_name | ipsec-upgrade |
+    Then the step should succeed
+    When I use the "ipsec-upgrade" project
+    Given I obtain test data file "networking/list_for_pods.json"
+    #Creating two test pods for pod-pod encryption check. Pods needs to be deployment/rc backed so that they can be migrate successfuly to the upgraded cluster.Creating each separat as they need to be on diff 
+    #nodes
+    When I run oc create over "list_for_pods.json" replacing paths:
+      | ["items"][0]["spec"]["template"]["spec"]["nodeName"]           | <%= cb.workers[1].name %> |
+      | ["items"][0]["spec"]["replicas"]                               | 1                         |
+      | ["items"][0]["metadata"]["name"]                               | hello-pod1-rc             |
+      | ["items"][0]["spec"]["template"]["metadata"]["labels"]["name"] | hello-pod1                |
+      | ["items"][1]["metadata"]["name"]                               | test-service1             |
+    Then the step should succeed
+    And a pod becomes ready with labels:
+      | name=hello-pod1 |
+    And evaluation of `pod.ip_url` is stored in the :test_pod_worker1 clipboard
+
+    Given I obtain test data file "networking/list_for_pods.json"
+    When I run oc create over "list_for_pods.json" replacing paths:
+      | ["items"][0]["spec"]["template"]["spec"]["nodeName"]                 | <%= cb.workers[0].name %>                                                                      |
+      | ["items"][0]["spec"]["replicas"]                                     | 1                                                                                              |
+      | ["items"][0]["metadata"]["name"]                                     | hello-pod0-rc                                                                                  |
+      | ["items"][0]["spec"]["template"]["metadata"]["labels"]["name"]       | hello-pod0                                                                                     |
+      | ["items"][1]["metadata"]["name"]                                     | test-service0                                                                                  |
+      | ["items"][0]["spec"]["template"]["spec"]["containers"][0]["command"] | ["bash", "-c", "for f in {0..36000}; do curl <%= cb.test_pod_worker1 %>:8080 ; sleep 1; done"] |
+
+    Then the step should succeed
+    #Above command will curl "hello openshift" traffic every 1 second to worker1 test pod which is expected to cause ESP traffic generation across those nodes
+    And a pod becomes ready with labels:
+      | name=hello-pod0 |
+      
```

Comment:
@anuragthehatter we better to add this kind of check steps to make sure this feature is working well before upgrade cluster
```
| sh | -c | timeout --preserve-status 2 tcpdump -i <%= cb.default_interface %> esp |
```

Reply:
@zhaozhanqi Yes. We can certainly do that as a pre-check to have a pre-upgrade datapoint. However normal regression tests will be covering such checks in our regular builds but anyways it would always good to have more datapoints on more builds :)

Reply:
Re-ran with above logic included
Pre- https://privatebin-it-iso.int.open.paas.redhat.com/?48d6d91beadc77fb#cN4ZF26BLxwCdaKaqEozTRMVqXwfzu5pywDgPUryC4V
Post- https://privatebin-it-iso.int.open.paas.redhat.com/?98b0c22cfed229e2#4XtSH6qRZ1kMvzTCsnEjb9xaapLNpwthngEKQJKNWc1r